private/buf/bufformat/formatter.go

```diff
@@ -116,11 +141,11 @@ func (f *formatter) SetPreviousNode(node ast.Node) {
 }
 
 // writeFile writes the file node.
-func (f *formatter) writeFile(fileNode *ast.FileNode) {
-	f.writeFileHeader(fileNode)
-	f.writeFileTypes(fileNode)
-	if fileNode.EOF != nil {
-		info := f.fileNode.NodeInfo(fileNode.EOF)
+func (f *formatter) writeFile() {
```

Comment:
I removed the argument because it wasn't necessary since the file node is a field on `f`. In fact, `writeFileHeader` and `writeFileTypes` were also passed this argument but never even used it. So this is just minor clean up.